api/volume.go

```diff
@@ -106,7 +107,15 @@ func (s *Server) VolumeCreate(rw http.ResponseWriter, req *http.Request) error {
 		return err
 	}
 
-	if volume.Frontend == "" {
+	// don't use strconv.ParseBool() here since volume.Standby can be empty.
+	var standby bool
+	if strings.Contains(volume.Standby, "true") {
```

Comment:
How about using `bool` in the type definition?

Reply:
If I use `bool`, there is no related input form in the UI page.

Reply:
UI?

Reply:
I means raw API in `<longhorn-url>/v1/`